### PR TITLE
ENT-3372: Renamed main package name from taxonomy-service to taxonomy-connector

### PR DESCRIPTION
**Pull request by saleem-latif**
__Description:__

After the update of the repository name from `taxonomy-service` to `taxonomy-connector`, we also need to update all the occurrences inside the repository. This PR handles these updates.

### CODE REVIEW

**Review by sameenfatima78 — APPROVED** [review]
Just one comment: I think you didn't make changes in this file -> https://github.com/edx/taxonomy-connector/blob/master/tests/test_apps.py#L3

otherwise LGTM!

File: docs/conf.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def get_version(*file_paths):
 # The name for this set of Sphinx documents.
 # "<project> v<release> documentation" by default.
 #
-# html_title = 'taxonomy-service v0.1.0'
+# html_title = 'taxonomy-connector v0.1.0'
```
Comment (moconnell1453) [inline docs/conf.py:185]: should the version be 1.1.0 to match CHANGELOG?

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=C0111,C0103
 """
-Setup configurations for taxonomy application.
+Setup configurations for the taxonomy application.
```
Comment (zamanafzal) [inline setup.py:4]: Don't we need to change the name here from taxonomy to taxonomy-connector?

**Review by moconnell1453 — APPROVED** [review]
I left one comment about the Sphinx version, otherwise it looks good to merge

**Comment by zamanafzal** [comment]
I think you forgot to update the name at 
tests>test_apps> L3.
Coverage.xml> L6

**Comment by saleem-latif** [comment]
@zamanafzal Coverage.xml is not tracked with git, I have made the other changes you suggested. please take another look

**Pull request merged**